### DISCUSSION TITLE
External Components cli installation

### DISCUSSION THREAD

**Discussion by beamitpal** [shadcn-ui/ui#4768] [General]
I want to share some of my reusable components how can I share that with npx shadcn@latest add https://example.com/components/component.json and please make a json  of dummy data so that I can understand properly. And also tell me what field is required and what is optional in the json dummy data .

**Comment by eluce2** [comment]
Take a look at this tool: https://github.com/jherr/shadcn-differ

**Reply by aryomuzakki** [reply]
can you help to just explain about the json that are required so I can skip using this tool? or is it really better just using the tool by making new shadcn init, adding new changes, then run the tool?
